config/application.php

```diff
@@ -60,6 +60,17 @@
 Config::define('DB_COLLATE', '');
 $table_prefix = env('DB_PREFIX') ?: 'wp_';
 
+if (env('DATABASE_URL')) {
```

Comment:
🤔 we require `DB_NAME', 'DB_USER', 'DB_PASSWORD'` when .env is loaded. We'd need to make that optional somehow wouldn't we?